### PR TITLE
Use Sass instead of Less

### PR DESCRIPTION
**Pull request by jpellizzari**
Needed for #2133 

Changes the Less files to Sass. We are standardizing around Sass on other projects.

@davkal and @bowenli Please review, specifically for styling regressions.

### CODE REVIEW

**Comment by bowenli** [comment]
looks fine to me

File: client/package.json [hunk 1]
```diff
@@ -60,19 +60,18 @@
...
     "file-loader": "0.9.0",
-    "font-awesome-webpack": "0.0.4",
     "html-webpack-plugin": "2.24.1",
     "http-proxy-rules": "1.0.1",
     "immutable-devtools": "0.0.7",
     "jest-cli": "17.0.3",
     "json-loader": "0.5.4",
-    "less": "2.7.1",
-    "less-loader": "2.2.3",
+    "node-sass": "^3.13.1",
```
Comment (davkal) [inline client/package.json]: Let's keep those versions fixed.

File: client/package.json [hunk 2]
```diff
@@ -60,19 +60,18 @@
...
     "postcss-loader": "1.2.0",
     "react-addons-perf": "15.4.1",
     "redux-devtools": "3.3.1",
     "redux-devtools-dock-monitor": "1.1.1",
     "redux-devtools-log-monitor": "1.1.1",
+    "sass-loader": "^4.1.1",
```
Comment (davkal) [inline client/package.json]: Let's keep those versions fixed.

File: client/webpack.local.config.js [hunk 3]
```diff
@@ -115,6 +111,10 @@ module.exports = {
         test: /\.jsx?$/,
         exclude: /node_modules|vendor/,
         loaders: ['babel']
+      },
+      {
+        test: /\.(scss|css)$/,
+        loader: 'style-loader!css-loader!sass-loader'
```
Comment (davkal) [inline client/webpack.local.config.js]: what happened to post-css?

**Comment by davkal** [comment]
Works well. Small issues:

* high contrast mode does not work (maybe different issue?)
* build warning
```
WARNING in ./~/xterm/dist/xterm.js
Critical dependencies:
1:481-488 This seems to be a pre-built javascript file. Though this is possible, it's not recommended. Try to require the original source to get better results.
 @ ./~/xterm/dist/xterm.js 1:481-488
```
* matched search text looks different:

Before
screen shot 2017-01-20 at 12 49 26

After
screen shot 2017-01-20 at 12 53 33

**Comment by rade** [comment]
> high contrast mode does not work

#2138? I am guessing this PR was branched prior to the fix.

**Comment by jpellizzari** [comment]
@davkal updated with the matched-text styles fixed, xterm warning suppressed, and post-css added back in.

**Comment by jpellizzari** [comment]
Ping @davkal please review after my changes.

File: client/app/styles/main.scss [hunk 4]
```diff
@@ -14,59 +18,60 @@
...
+$link-opacity-default: 0.8;
+
+$search-border-color: transparent;
+$search-border-width: 1px;
 
 /* specific elements */
-@body-background-color: linear-gradient(30deg, @background-color 0%, @background-lighter-color 100%);
-@label-background-color: fade(@background-average-color, 70%);
+$body-background-color: linear-gradient(30deg, $background-color 0%, $background-lighter-color 100%);
+$label-background-color: fade-out($background-average-color, .7);
```
Comment (foot) [inline client/app/styles/main.scss]: change to `0.3`

File: client/app/styles/main.scss [hunk 5]
```diff
@@ -191,14 +196,14 @@ h2 {
   position: absolute;
   bottom: 11px;
   right: 43px;
-  color: @text-tertiary-color;
-  background-color: fade(@background-average-color, 90%);
+  color: $text-tertiary-color;
+  background-color: fade-out($background-average-color, .9);
```
Comment (foot) [inline client/app/styles/main.scss]: `0.1`

File: client/app/styles/main.scss [hunk 6]
```diff
@@ -1508,14 +1517,14 @@ h2 {
...
-    border-color: @weave-blue;
+    border-color: $weave-blue;
   }
 
 }
 
 .search-item {
-  background-color: fade(@weave-blue, 20%);
-  border-radius: @border-radius / 2;
+  background-color: fade-out($weave-blue, .2);
```
Comment (foot) [inline client/app/styles/main.scss]: `0.8`

**Comment by foot** [comment]
LGTM!

**Pull request merged**